packages/@react-spectrum/steplist/test/StepList.test.tsx

```diff
@@ -214,8 +217,8 @@ describe('StepList', function () {
       </StepList>
     );
 
-    expect(onLastCompletedStepChange).toHaveBeenCalledTimes(0);
-    expect(stepListItems[0].firstElementChild.textContent).toContain('Completed');
+    expect(onLastCompletedStepChange).toHaveBeenCalledWith('step-one');
```

Comment:
Hey @snowystinger thanks for contributing! You've added an assertion here to check `onLastCompletedStepChange` has been fired with `step-one` but the component was given a `defaultLastCompletedStep="step-one"` prop and the prop hasn't changed. It should only change once step-two has completed and step-two only completes once the selected key has advanced to step-three.

Reply:
After reading https://github.com/adobe/react-spectrum/issues/5013#issuecomment-1703101568

Are you suggesting it should be called after the initial render? Similar to `onSelectionChange`?

Reply:
sorry, missed the default or it my mind break, you are correct, it should not be called until after we've advanced it to step two